README.md

```diff
@@ -32,12 +32,25 @@ It's unlikely you will want to use the packages provided by your system (e.g. fr
 
 The scripts assume you can run the `docker` command WITHOUT sudo.
 
-NOTE: We are using features that require you to have at least docker 1.10 and docker-compose 1.6 installed.
+**NOTE:** We are using features that require you to have at least docker 1.10 and docker-compose 1.6 installed.
 
 ### Docker Compose
 Mac and Windows: Docker compose will have been installed as part of the docker toolbox.
 Linux: Follow the instructions to install the latest docker-compose [here] (https://github.com/docker/compose/releases)
 
+### Register at DockerHub
+
+The project uses private Docker repositories for certain images and to access these you will need to register at [DockerHub](https://hub.docker.com). Once you have registered, give your
```

Comment:
Can you use the "one sentence per line" pattern? It makes it much easier to review future changes and minimises merge-conflicts.